python/huggingfaceserver/huggingfaceserver/model.py

```diff
@@ -98,6 +98,7 @@ def __init__(
         self.vllm_engine_args = engine_args
         self.use_vllm = not kwargs.get("disable_vllm", False) if _vllm else False
         self.ready = False
+        self.dtype = kwargs.get("dtype","bfloat16")
```

Comment:
Typically BF16 is for training, for inference FP16 is more acceptable

Reply:
is this the argument defined from vllm ?

Reply:
Yes it is defined [here](https://arc.net/l/quote/pouzgdju). I verified the dtype of the model after engine creation and it matches with the input.
<img width="788" alt="image" src="https://github.com/kserve/kserve/assets/39181234/833c3ab1-abab-4c9f-babd-9b0432875e56">

Reply:
Can we add a comment that this argument is used by both HF and vllm?

Reply:
also the default should be float16 ?

Reply:
`auto` defaults to FP16 even if the model is FP32 in case of vLLM
but for HF, it doesn't. It sticks to FP32. And HF config doesn't have the info about what type the model weights are of. So for FP32 models (which are rare but do exist. GPTJ, OLMo etc for eg), this would again cause inconsistency.